CONTRIBUTING.md

```diff
@@ -28,3 +28,49 @@ Members of the Contributors team are encouraged to review pull requests that
 have already been reviewed, and pull request contributors are encouraged to seek
 multiple reviews.  Reviews from anyone not on the Contributors team are always
 appreciated and encouraged!
+
+## Reviewing Pull Requests
+
+All mlpack contributors who choose to review and provide feedback on Pull
```

Comment:
I think you missed a place to make it `pull requests`

Reply:
woooh, That was a nice catch, made changes